addons/report/models/abstract_report.py

```diff
@@ -1,37 +1,33 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp.osv import osv
+from odoo import api, models
 
 
-class AbstractReport(osv.AbstractModel):
+class AbstractReport(models.AbstractModel):
     """Model used to embed old style reports"""
     _name = 'report.abstract_report'
     _template = None
     _wrapped_report_class = None
 
-    def render_html(self, cr, uid, ids, data=None, context=None):
-        context = dict(context or {})
-
+    @api.multi
+    def render_html(self, data=None):
+        context = dict(self.env.context or {})
         # If the key 'landscape' is present in data['form'], passing it into the context
         if data and data.get('form', {}).get('landscape'):
             context['landscape'] = True
-
-        if context and context.get('active_ids'):
+        if context.get('active_ids'):
             # Browse the selected objects via their reference in context
             model = context.get('active_model') or context.get('model')
-            objects_model = self.pool[model]
-            objects = objects_model.browse(cr, uid, context['active_ids'], context=context)
+            objects = self.env[model].browse(self.ids)
```

Comment:
why not context['active_ids'] instead of self.ids ? 
are you sure is it working  self.env[model].browse(self.ids)?